pyroute2/ipset.py

```diff
@@ -161,12 +163,24 @@ def create(self, name, stype='hash:ip', family=socket.AF_INET,
         if forceadd:
             cadt_flags |= IPSET_FLAG_WITH_FORCEADD
 
+        if 'bitmap' in stype and bitmap_range is None:
```

Comment:
I think that it's probably a good idea to split stype here and check that first element is equal to "bitmap"